load/scan.go

```diff
@@ -5,10 +5,10 @@ import (
 	"reflect"
 )
 
-// ackAndMaybeSend adjust the outstanding batches count and potentially sends
-// another batch to the worker via ch. If unsent is non-empty, send. Returns
-// the updated state of unsent
-func ackAndMaybeSend(ch *duplexChannel, count *int, unsent []Batch) []Batch {
+// sendOutstandingBatchToWorker adjust the outstanding batches count and potentially sends
```

Comment:
Not a fan of this name change. Less accurate than before.

Reply:
Let's just call it `sendBatch`, so there will be two functions in charge of sending batches:
1. `func sendBatch(ch *duplexChannel, count *int, unsent []Batch)`
2. `func sendOrQueueBatch(ch *duplexChannel, count *int, batch Batch, unsent []Batch)`

and I am not sure about correctness of `	*count--` and `*count++` being outside of `if`s in those functions